src/cruds/curds.js

```diff
@@ -0,0 +1,56 @@
+class Cruds {
+  constructor() {
+    this.tasks = JSON.parse(window.localStorage.getItem('NOTE') || '[]');
+  }
+
```

Comment:
Hi. In order to load the localStorage, I don't think we need to use window to call localStorage. i.e. localStorage.getItem() should be enough.

Reply:
thanks to mention and notice me, I will remove the window to call local storage because localStorage.getItem() should work. properly.